filepath.go

```diff
@@ -91,24 +91,14 @@ func LineCnt(fileName string) int {
 
 // LineCntr counts no. of lines for given reader
 func LineCntr(r io.Reader) int {
-	buf := make([]byte, 8196)
-	count := 0
-	lineSep := []byte{'\n'}
-
-	for {
-		c, err := r.Read(buf)
-		if err != nil && err != io.EOF {
-			return count
-		}
 
-		count += bytes.Count(buf[:c], lineSep)
+	contents, err := ioutil.ReadAll(r)
```

Comment:
When I get a time, I will optimize it further. Currently it does acceptable performance without loading entire file into memory.